.travis.yml

```diff
@@ -51,19 +50,14 @@ before_script:
   #   export MOI_CONFIG_FP=`pwd`/qiita_core/support_files/config_test_travis.cfg;
   #   fi
   - qiita-env make --no-load-ontologies
-  - |
-    if [ ${TEST_ADD_STUDIES} == "False" ]; then
-        qiita pet webserver --no-build-docs start &
-    fi
+  - qiita-test-install
 script:
   - sleep 5
   - if [ ${TEST_ADD_STUDIES} == "True" ]; then test_data_studies/commands.sh ; fi
   - if [ ${TEST_ADD_STUDIES} == "True" ]; then qiita-cron-job ; fi
-  - if [ ${TEST_ADD_STUDIES} == "False" ]; then qiita-test-install ; fi
-  - if [ ${TEST_ADD_STUDIES} == "False" ]; then nosetests $COVER_PACKAGE --with-doctest --with-coverage --with-timer -v --cover-package=$COVER_PACKAGE; fi
+  - qiita pet webserver --no-build-docs start &
+  - nosetests $COVER_PACKAGE --with-doctest --with-coverage --with-timer -v --cover-package=$COVER_PACKAGE
   - flake8 qiita_* setup.py scripts/*
-  - ls -R /home/travis/miniconda3/envs/qiita/lib/python2.7/site-packages/qiita_pet/support_files/doc/
-  - qiita pet webserver
```

Comment:
Can you re-add this command? It is the one that checks that the documentation builds correctly.